db/migrate/20160401043112_create_phones.rb

```diff
@@ -5,7 +5,7 @@ def change
 
       t.string :number, null: false
       t.string :extension
-      t.string :type, null: false
+      t.string :service_type, null: false
```

Comment:
Well the migration works fine so would we want to handle this in that way at this point?

Just to avoid "custom" configurations, I don't think there was any specific desire to have it called specifically "type"